tox.ini

```diff
@@ -47,6 +47,12 @@ deps = -e.[dev]
 commands =
     mypy gql tests
 
+[testenv:docs]
+basepython = python3.8
+deps = -e.[dev]
+commands =
+    sphinx-build -b html -nEW docs docs/_build/html
```

Comment:
If I run this command I get the error:
```Warning, treated as error:
/home/leszek/miniconda3/envs/gql-dev/lib/python3.8/site-packages/gql/client.py:docstring of gql.client.AsyncClientSession._execute::py:class référence cible non trouvée : graphql.language.ast.DocumentNode
```

But the documentation is correctly generated if I run `make html` in the docs folder

Reply:
I already asked Cito on the Slack channel since I have no idea why those are generated. Seems like `graphql-core` has something similar and then nitpicking each one on the `conf.py` file. I tried the same but always got that error.

Reply:
@leszekhanusz run again but without the `n`:

`sphinx-build -b html -nEW docs docs/_build/html` Now these errors are gone!